test/unit/reductions/grid_search/test_grid_search_arguments.py

```diff
@@ -350,14 +350,33 @@ def test_bgl_gpa_data(self):
 
         names = ['gender', 'physics', 'biology', 'history', 'second_language',
                  'geography', 'literature', 'portuguese', 'math', 'chemistry', 'gpa']
-        data = pd.DataFrame(np.array([[0, 622.6, 491.56, 439.93, 707.64, 663.65, 557.09, 711.37, 731.31, 509.8, 1.33333],
-        [1, 538, 490.58, 406.59, 529.05, 532.28, 447.23, 527.58, 379.14, 488.64, 2.98333],
-        [1, 455.18, 440, 570.86, 417.54, 453.53, 425.87, 475.63, 476.11, 407.15, 1.97333],
-        [0, 756.91, 679.62, 531.28, 583.63, 534.42, 521.4, 592.41, 783.76, 588.26, 2.53333],
-        [1, 584.54, 649.84, 637.43, 609.06, 670.46, 515.38, 572.52, 581.25, 529.04, 1.58667],
-        [1, 325.99, 466.74, 597.06, 554.43, 535.77, 717.03, 477.6, 503.82, 422.92, 1.66667],
-        [0, 622.6, 587.04, 598.85, 603.32, 690.7, 652.86, 533.05, 755.3, 628.73, 3.72333],
-        [1, 527.65, 559.99, 758.37, 669.71, 645.62, 648.67, 539.23, 470.78, 486.13, 3.08333]]), columns=names)
+        data = pd.DataFrame(np.array([[0, 622.6, 491.56, 439.93, 707.64, 663.65, 557.09, 711.37,
```

Comment:
whenever the flake8 / pep8 rules are in the way of readability we can make exceptions. Inline that would be
```
<offending line>  # noqa: A123
```
with exactly two whitespaces after the code, and the abbreviation is the offending pep8 code (usually a letter followed by three digits)

If you don't want to do this in this PR that's fine, I can take care of it in my follow-up as well